packages/react-scripts/config/zion-webpack.config.js

```diff
@@ -0,0 +1,50 @@
+require('dotenv').config()
+
+const initiatedDirectory = process.env.INIT_CWD
+const nodeExternals = require('webpack-node-externals')
+const merge = require('webpack-merge')
+const setupProxy = require('../proxy/setupProxy')
+const baseWebpack = require('./webpack.config.js')
+
+const craWebpack = baseWebpack('development')
+delete craWebpack.entry
+delete craWebpack.output
+delete craWebpack.plugins
+delete craWebpack.optimization
+
+module.exports = merge(craWebpack, {
+  externals: [nodeExternals(), 'react', 'react-dom', /^@fs/, /^@fs-sandbox/],
+  entry: `${initiatedDirectory}/src/index.js`,
+  output: {
+    path: `${initiatedDirectory}/dist`,
+    filename: chunkData => (chunkData.chunk.name === 'main' ? 'index.js' : '[name].js'),
+    libraryTarget: 'commonjs2',
+  },
+  resolve: {
+    alias: {
+      // fix duplicate copies of react issue for hooks
+      // https://github.com/webpack/webpack/issues/8607
+      react: require.resolve('react'),
+      'react-dom': require.resolve('react-dom'),
+    },
+  },
+  // TODO: we can probably remove the whole module prop once babel-preset-frontier gets into react-scripts
```

Comment:
has this happened yet?

Reply:
I thought it had, but when I remove this, it fails to transpile.

Reply:
@joeycozza do you know where we are on this?

Reply:
Exactly where Tyler is. The code in react-scripts/config/webpack.config.js does have a rule using @fs/babel-preset-frontier, BUT there are more configurations/options set in that one than there are here in this zion-webpack.config.

So I assume that some of those options aren't good for what zion wants/needs.

Reply:
So do we remove the TODO? Any idea who would know the final status on this?

Reply:
I don't think there is a final status. We could either do some webpack and babel deep dive to see why removing this here breaks zion, or we could just remove the TODO and let this be.

Reply:
I think we should dive into it when we can. I'll create a card.